Cargo.toml

```diff
@@ -7,5 +7,3 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-lazy_static = "1.4.0"
```

Comment:
please do not remove dependencies, we need them in the future

Reply:
ok